Products/CMFPlone/patterns/tinymce.py

```diff
@@ -100,12 +100,13 @@ def get_all_style_formats(self):
         }, {
             'title': 'Alignment',
             'items': self.get_styles(alignment_styles)
-        }, {
+        },{
```

Comment:
I do not want to be annoying but here we still have a missing space...

Reply:
Not at all. Will take care of the issue.